javalib/src/main/scala/java/time/DayOfWeek.scala

```diff
@@ -0,0 +1,79 @@
+package java.time
+
+import java.time.temporal._
+
+final class DayOfWeek private (name: String, ordinal: Int)
```

Comment:
Should use the `value` from `getValue()` rather than `ordinal` to avoid the `+ 1` each time it is used.